src/keytar_win.cc

```diff
@@ -228,6 +228,9 @@ KEYTAR_OP_RESULT FindCredentials(const std::string& service,
                                  std::vector<Credentials>* credentials,
                                  std::string* errStr) {
   LPWSTR filter = utf8ToWideChar(service + "*");
+  if (filter == NULL) {
+    return FAIL_ERROR;
```

Comment:
Probably worth adding an error message for this?
```suggestion
    *errStr = "Error generating credential filter";
    return FAIL_ERROR;
```